### ISSUE TITLE
Uncaught Error: Java exception was raised during method invocation", source: file:///android_asset/js/Bridge.js 

### ISSUE THREAD

**Issue by Nikhil-z** [FolioReader/FolioReader-Android#242]
I have found some issues with 0.4.0 version of folio reader. I was trying to open a file from phone storage, the file gets open correctly but the loader keep on spinning till user touch the screen where loader get dismissed and content will be displayed

Following log was generated

```
07-17 16:58:19.477 31310-31310/com.myapp I/chromium: [INFO:CONSOLE(881)] "Uncaught Error: Java exception was raised during method invocation", source: file:///android_asset/js/Bridge.js (881)
07-17 16:58:19.505 31310-32115/com.myapp V/FolioWebView: -> setCompatMode -> compatMode = CSS1Compat
07-17 16:58:19.513 31310-31310/com.myapp I/chromium: [INFO:CONSOLE(373)] "-> scrollToLast", source: file:///android_asset/js/Bridge.js (373)
07-17 16:58:19.540 31310-32115/com.myapp W/System.err: android.view.ViewRootImpl$CalledFromWrongThreadException: Only the original thread that created a view hierarchy can touch its views.
        at android.view.ViewRootImpl.checkThread(ViewRootImpl.java:6957)
        at android.view.ViewRootImpl.invalidateChildInParent(ViewRootImpl.java:1124)
        at android.view.ViewGroup.invalidateChild(ViewGroup.java:5207)
        at android.view.View.invalidateInternal(View.java:13679)
        at android.view.View.invalidate(View.java:13643)
        at android.view.View.setFlags(View.java:11550)
        at android.view.View.setVisibility(View.java:8082)
07-17 16:58:19.541 31310-32115/com.myapp W/System.err:     at com.folioreader.view.LoadingView.hide(LoadingView.java:94)
        at org.chromium.base.SystemMessageHandler.nativeDoRunLoopOnce(Native Method)
        at org.chromium.base.SystemMessageHandler.handleMessage(PG:9)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:163)
        at android.os.HandlerThread.run(HandlerThread.java:61)
```

**Comment by hrishikesh-kadam** [comment]
This has been fixed in https://github.com/codetoart/FolioReader-Android/commit/8fc350eece7dfa6ec933528287b44bd663edf9de. Changes required to address this issue is of just LoadingView.java

This fix will get released in proposed v0.4.1

**Comment by hrishikesh-kadam** [comment]
@Nikhil-z v0.4.1 has been released. Thanks for reporting the issue.

@mahavir155 We can close this issue.
